### DISCUSSION TITLE
ImageMagick Voluntary Product Accessibility Template (VPAT)

### DISCUSSION THREAD

**Discussion by ben-mur** [ImageMagick/ImageMagick#4461] [General] [answered]
Hello everyone, I'm not sure where to direct this question, but we all using ImageMagick and our organization has asked for a Voluntary Product Accessibility Template (VPAT) ( https://www.section508.gov/sell/vpat/) to be requested or completed for it's continued use. Knowing that a VPAT likely does not exist, does the community have any information on the accessibility of ImageMagick I could use as a substitute? Any help would be appreciated!

**Comment by urban-warrior** [comment, marked as answer]
ImageMagick is an open-source command-line utility that provides basic accessibility features for users who rely on assistive technologies. However, the software is not optimized for use with assistive technologies and may require significant customization to be accessible to all users. The project provides accessible online documentation, but may not be suitable for non-technical users.

See https://imagemagick.org/script/vpat.php.
